nav2_amcl/src/amcl_node.cpp

```diff
@@ -878,9 +705,9 @@ AmclNode::laserReceived(sensor_msgs::msg::LaserScan::ConstSharedPtr laser_scan)
         tmp_tf_stamped.header.stamp = laser_scan->header.stamp;
         tf2::toMsg(tmp_tf.inverse(), tmp_tf_stamped.pose);
 
-        this->tf_->transform(tmp_tf_stamped, odom_to_map, odom_frame_id_);
+        tf_buffer_->transform(tmp_tf_stamped, odom_to_map, odom_frame_id_);
       } catch (tf2::TransformException) {
-        RCLCPP_DEBUG(get_logger(), "Failed to subtract base to odom transform");
+        RCLCPP_INFO(get_logger(), "Failed to subtract base to odom transform");
```

Comment:
Should this be an INFO message as opposed to DEBUG or WARN?

Reply:
Sure, I've changed to back to DEBUG. I was probably using INFO while debugging, but forgot to change it back.